Example/Example/RootWireframe.swift

```diff
@@ -23,12 +23,28 @@ class RootWireframe: ExampleWireframe {
         let charactersNavigationController = serviceLocator.provideCharactersNavigationController()
         serviceLocator.navigatorContainer.register(charactersNavigationController)
 
-        let comicsNavigationController = serviceLocator.provideComicsNavigationController()
-        ServiceLocator.sharedInstance.navigatorContainer.register(comicsNavigationController)
+        let seriesNavigationController = serviceLocator.provideSeriesNavigationController()
+        ServiceLocator.sharedInstance.navigatorContainer.register(seriesNavigationController)
 
         return [
             charactersNavigationController,
-            comicsNavigationController
+            seriesNavigationController
         ]
     }
+
+    private func configureTabBarItems(tabBar: UITabBar) {
+        tabBar.accessibilityLabel = "MainWireframe TabBar"
+        tabBar.tintColor = UIColor.tabBarTintColor
+
+        let charactersIcon = UIImage(named: "tab_bar_icon_characters")
```

Comment:
I'v tried but the view controllers inside the tab bar view controllers are configured programmatically and I have no options to add or modify a tab bar item :(